functions-and-scope.js

```diff
@@ -57,8 +102,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
+function avarageGrade(arr) {
+    let total = 0;
+    let totalGrades = 0;
+    for (let i = 0; i < arr.length; i++) {
+        total += arr[i]
+        totalGrades++;
+    }
+    let avarage = total / totalGrades
+    return Math.round(avarage * 100) / 100;
```

Comment:
Hier zou je perfect de toFixed functie voor kunnen gebruiken
```suggestion
    return avarage.toFixed(2);
```